app/src/main/java/org/p2p/wallet/splash/SplashPresenter.kt

```diff
@@ -41,4 +57,55 @@ class SplashPresenter(
             view?.navigateToOnboarding()
         }
     }
+
+    private suspend fun handleDeeplink() {
+        deeplinksManager.pendingDeeplinkUri?.let { deeplinkUri ->
+            deeplinkUri.getQueryParameter(DeeplinkQuery.value)?.let { seed ->
+                if (seed.isNotEmpty()) {
+                    val keys = seed.split("-").map {
+                        SeedPhraseWord(
+                            text = it,
+                            isValid = true,
+                            isBlurred = false
+                        )
+                    }
+                    verifySeedPhrase(keys, deeplinkUri)
+                    return
+                }
+            }
+        }
+        openRootScreen()
+    }
+
+    private suspend fun verifySeedPhrase(seedPhrase: List<SeedPhraseWord>, deeplink: Uri) {
+        var currentSeedPhrase = seedPhrase
```

Comment:
Why did you do it? 
I think its' redundant

Reply:
wow sorry it was in code but after i remove some lines yes it is redundant =) lol